### PR TITLE
add check for []interface{} type when validating audience

### PR DESCRIPTION
**Pull request by yehted**
Addresses #7

### CODE REVIEW

File: map_claims.go [hunk 1]
```diff
@@ -13,15 +13,27 @@ type MapClaims map[string]interface{}
...
+			aud = append(aud, a.(string))
+		}
+	case string:
 		strAud, ok := m["aud"].(string)
 		if !ok {
 			return false
 		}
 		aud = append(aud, strAud)
+	default:
+		return false
```
Comment (johnweldon) [inline map_claims.go]:
```suggestion
	var aud []string
	switch v := m["aud"].(type) {
	case []string:
		aud = v
	case []interface{}:
		for _, a := range v {
			vs, ok := a.(string)
			if !ok {
				return false
			}
			aud = append(aud, vs)
		}
	case string:
		strAud, ok := v.(string)
		if !ok {
			return false
		}
		aud = append(aud, strAud)
	default:
		return false
```

Comment (yehted) [reply map_claims.go]: line 44 doesn't compile. line 48 is sufficient. deleted lines 44-47

**Review by softwarebygabe — DISMISSED** [review]
👍

**Review by Waterdrips — DISMISSED** [review]

**Comment by Waterdrips** [comment]
@yehted 
` The base branch requires all commits to be signed. Learn more about signing commits. `
https://docs.github.com/articles/about-gpg/

If you want to sign the commits I can merge

**Review by paulaolmedo — APPROVED** [review]

**Comment by viglia** [comment]
@Waterdrips any chance to merge this PR please?

thank you!

**Review by ah-f3 — APPROVED** [review]

**Comment by ah-f3** [comment]
@viglia New release for you https://github.com/form3tech-oss/jwt-go/releases/tag/v3.2.3

v3.2.3

**Comment by viglia** [comment]
@ah-f3 many thanks! 👍🏻  :D

**Pull request merged**